### PR TITLE
[enhancement](transatcion) show more details about transaction num when execute show proc "/dbs" stmt

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

1. add transactionNum in dbs proc dir
2. show more details about transaction num in table proc dir when execute show "/dbs/dbId" stmt
original:
```
mysql> show proc "/dbs/10007";
+---------+-----------+----------+---------------------+--------------+--------+------+--------------------------+--------------+----------------+--------------------+---------------------+
| TableId | TableName | IndexNum | PartitionColumnName | PartitionNum | State  | Type | LastConsistencyCheckTime | ReplicaCount | VisibleVersion | VisibleVersionTime | LastUpdateTime      |
+---------+-----------+----------+---------------------+--------------+--------+------+--------------------------+--------------+----------------+--------------------+---------------------+
| 12289   | t1        | 1        | NULL                | 1            | NORMAL | OLAP | NULL                     | 1            | 1              | 1707381400540      | 2024-02-08 16:36:40 |
+---------+-----------+----------+---------------------+--------------+--------+------+--------------------------+--------------+----------------+--------------------+---------------------+
1 row in set (0.00 sec)
```
now:
```
mysql> show proc "/dbs/10007";
+---------+-----------+----------+---------------------+--------------+--------+------+--------------------------+--------------+----------------+--------------------+-----------------------+---------------------+
| TableId | TableName | IndexNum | PartitionColumnName | PartitionNum | State  | Type | LastConsistencyCheckTime | ReplicaCount | VisibleVersion | VisibleVersionTime | RunningTransactionNum | LastUpdateTime      |
+---------+-----------+----------+---------------------+--------------+--------+------+--------------------------+--------------+----------------+--------------------+-----------------------+---------------------+
| 12289   | t1        | 1        | NULL                | 1            | NORMAL | OLAP | NULL                     | 1            | 1              | 1707381400540      | 0                     | 2024-02-08 16:36:40 |
+---------+-----------+----------+---------------------+--------------+--------+------+--------------------------+--------------+----------------+--------------------+-----------------------+---------------------+
1 row in set (0.00 sec)

```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by caiconghui** [comment]
run buildall

**Comment by caiconghui** [comment]
run buildall

**Comment by caiconghui** [comment]
run buildall

**Comment by caiconghui** [comment]
run buildall

**Comment by caiconghui** [comment]
run buildall

**Comment by caiconghui** [comment]
run pipelinex_p0

**Comment by caiconghui** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/proc/DbsProcDir.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ public class DbsProcDir implements ProcDirInterface {
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("DbId").add("DbName").add("TableNum").add("Size").add("Quota")
             .add("LastConsistencyCheckTime").add("ReplicaCount").add("ReplicaQuota")
-            .add("TransactionQuota").add("LastUpdateTime")
+            .add("TransactionNum").add("TransactionQuota").add("LastUpdateTime")
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/proc/DbsProcDir.java:46]:
```suggestion
            .add("RunningTransactionNum").add("TransactionQuota").add("LastUpdateTime")
```

**Comment by caiconghui** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Review by weizuo93 — DISMISSED** [review]

**Comment by caiconghui** [comment]
run pipelinex_p0

**Review by morningman — APPROVED** [review]

**Review by weizuo93 — APPROVED** [review]

**Pull request merged**